03-transactions.md

```diff
@@ -139,6 +179,25 @@ This output sends funds to either an HTLC-timeout transaction after the HTLC-tim
         OP_ENDIF
     OP_ENDIF
 
+Or, with `option_anchor_outputs`:
+
+    # To remote node with revocation key
+    OP_DUP OP_HASH160 <RIPEMD160(SHA256(revocationpubkey))> OP_EQUAL
+    OP_IF
+        OP_CHECKSIG
+    OP_ELSE
+        1 OP_CHECKSEQUENCEVERIFY OP_DROP
```

Comment:
Same comment here, this should be moved to reside within the internal else clause. We don't need to further encumber the second-level redemption path as there's only one way that can be spent: with a single transaction sending the funds to a csv output.

Reply:
Yes, clear. Resolving this comment for overview and tracking in previous one